### ISSUE TITLE
Gaussian Process (GP) Support

### ISSUE THREAD

**Issue by 392781** [intel/scikit-learn-intelex#928]
Will `intelex` ever support `scikit-learn` gaussian process regressors/classifiers?

**Comment by agorshk** [comment]
> Will `intelex` ever support `scikit-learn` gaussian process regressors/classifiers?

Hi,
Thanks for the question. Right now intelex doesn't support gaussian process regressors/classifiers. Please provide more details of your task if it's possible (notebook or some description). It helps to understand what problems you try to solve and prioritize our work.

**Comment by 392781** [comment]
No, I am aware that it's not supported, hence my original query.  I was
wondering if GP support is something that is slated to be included in the
future for intelex.

**Comment by agorshk** [comment]
We don't have plans to implement it in the first half of 2022
